datahub-web-react/src/app/entity/shared/components/styled/DeprecationPill.tsx

```diff
@@ -131,14 +155,53 @@ export const DeprecationPill = ({ deprecation, urn, refetch, showUndeprecate }:
 
     return (
         <Popover
-            overlayStyle={{ maxWidth: 240 }}
+        overlayStyle={{ maxWidth: 480 }}
             placement="right"
             content={
                 hasDetails ? (
                     <>
                         {deprecation?.note !== '' && <DeprecatedTitle>Deprecation note</DeprecatedTitle>}
                         {isDividerNeeded && <ThinDivider />}
-                        {deprecation?.note !== '' && <DeprecatedSubTitle>{deprecation.note}</DeprecatedSubTitle>}
+                        <DescriptionContainer>
+                            {expanded || !overLimit ? (
+                                <>
+                                    {deprecation?.note && deprecation?.note !== '' && <StyledViewer content={deprecation.note} readOnly />}
```

Comment:
This makes sense logically but it can be simplified I feel. There is too much nesting and multiple conditionals happening. 

Is there any way to simplify this condition?